Jenkinsfile

```diff
@@ -1,68 +1,32 @@
 #!/usr/bin/env groovy
-// vim: set ts=4 sts=4 sw=4 noet:
+
+@Library('lco-shared-libs@0.0.10') _
 
 pipeline {
 	agent any
 	environment {
-		DOCKER_IMG = "docker.lco.global/pupe-pat"
-		GIT_COMMIT_SHORT_ID = getCommit().take(12)
-		GIT_TAG_NAME = gitTagName()
+		dockerImage = null
+		PROJ_NAME = projName()
+		GIT_DESCRIPTION = gitDescribe()
+		DOCKER_IMG = dockerImageName("${LCO_DOCK_REG}", "${PROJ_NAME}", "${GIT_DESCRIPTION}")
+	}
+	options {
+		timeout(time: 1, unit: 'HOURS')
 	}
 	stages {
-		stage('Build Docker image') {
+		stage('Build image') {
 			steps {
 				script {
-					dockerImage = docker.build("${DOCKER_IMG}:${GIT_COMMIT_SHORT_ID}", "--pull .")
+					dockerImage = docker.build("${DOCKER_IMG}")
 				}
 			}
 		}
-		stage('Push Docker image') {
+		stage('Push image') {
 			steps {
 				script {
-					// always push the Docker image with the git commit version
-					dockerImage.push()
-
-					if (env.BRANCH_NAME == 'master') {
-						// this commit has a git tag associated, push the Docker image
-						// with the git tag name as the Docker image tag name
-						if (env.GIT_TAG_NAME != null && env.GIT_TAG_NAME != "null") {
-							dockerImage.push(env.GIT_TAG_NAME)
-						}
-
-						// the master branch always updates the Docker tag "latest" for
-						// user convenience purposes
-						dockerImage.push('latest')
-					}
+					dockerImage.push("${GIT_DESCRIPTION}")
```

Comment:
`${GIT_DESCRIPTION}` doesn't need to be specified here because it's in the full imaged name created on line 11.